style.css

```diff
@@ -190,12 +103,11 @@ h1 {
 
 p {
     line-height: 40px;
-    font-size: 20px;
-     
+    font-size: 20px;    
 }
 
 button {
```

Comment:
You need to use classes for this instead of the name of the tag. what if you have more than one button and they have different styles?